### ISSUE TITLE
Why did you pull passive-endermen v2.3 for forge?

### ISSUE THREAD

**Issue by PongoSapiens** [ricksouth/serilum-mc-mods#1216]
## **Information**
Minecraft version: 1.18.2
Modloader: Forge / Fabric
Forge version: 40.1.64
Environment: Multiplayer

Mod name: Passive Endermen (Forge)
Mod version: That is the issue.

I run a server for myself and some friends, and we are using Passive Endermen to prevent endermen griefing. Version 2.3 claims to fix a bug where Endermen could still pick up blocks, and I thought that was working. Then you pulled version 2.3 leaving only 2.1 with no explanation anywhere as to why, should I downgrade, what do I risk by not downgrading? A mod to prevent enderman griefing that doesn't do so is not super useful. Anyway without any indication as to why you pulled 2.3 I'm stuck, am I risking corrupting my server?

**Comment by Serifina** [comment]
I would also like to know. My modpack on CurseForge uses the 2.3 version (as 2.1 really did not stop endermen from griefing). With 2.3 pulled, I can't update my modpack without being forced to downgrade the file, and I really would prefer not to have to go to a datapack solution for this problem.

**Comment by PongoSapiens** [comment]
I'd like to add that I'm not suggesting you shouldn't have pulled it, just that you should have told us why you did. I assume you had a good reason for pulling it, but without knowing what that reason was, I can't evaluate the risks to my server.

**Comment by ricksouth** [comment]
Hi guys, appreciate the interest. I archived the other versions, as I had multiple messages about them not working correctly. I wasn't able to fix it at that time, so this way I thought to prevent others from having the same issue.

That being said, I just released version 3.0 which solves the previously mentioned issues. Feel free to update to that. Any world will be without risks, with and without the mod installed.

https://www.curseforge.com/minecraft/mc-mods/passive-endermen
